nordicsemi/zigbee/ota_file.py

```diff
@@ -48,6 +48,10 @@
 OTA_UPGRADE_SUBELEMENT_HEADER_SIZE    = 2 + 4  # Tag ID + Length Field
 OTA_UPGRADE_SUBELEMENT_TRIGGER_TYPE   = 0xCDEF
 OTA_UPGRADE_SUBELEMENT_TRIGGER_LENGTH = 1 + 4 + 4 + 4 + 4 # See background_dfu_trigger_t in the nRF5 SDK for explanation
+
+OTA_UPGRADE_FIELD_CONTROL_WITH_HW_VER = (1 << 2)
```

Comment:
Just a thought: what do you think about placing "bit mask" inside the name of this constant?